cases/views.py

```diff
@@ -138,8 +139,8 @@ def reassign(request, pk):
     if form.is_valid():
         form.save()
         user = form.cleaned_data["assigned"]
-        case.followers.add(user)
-        case.notify_followers(f"Assigned {user}.", triggered_by=request.user)
+        case.assign(user, triggered_by=request.user)
+        case.save()
```

Comment:
Do we need this save? The form got saved above (which presumably updated assigned) and the assign() function doesn't change anything.